### DISCUSSION TITLE
ProxSuite 0.5.0

### DISCUSSION THREAD

**Discussion by jcarpent** [Simple-Robotics/proxsuite#263] [Announcements]
This release adds [**support for nonconvex QPs**](https://github.com/Simple-Robotics/proxsuite/issues/237), along with healthy fixes.

## What's Changed
* Fix compilation (veg/memory) for gcc 7 and clang 7 by @costashatz in https://github.com/Simple-Robotics/proxsuite/pull/255
* Estimate minimal eigenvalue of quadratic cost hessian by @Bambade in https://github.com/Simple-Robotics/proxsuite/pull/257
* Fix typo #254  by @Bambade in https://github.com/Simple-Robotics/proxsuite/pull/258
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/Simple-Robotics/proxsuite/pull/260
* Sync submodule cmake by @jcarpent in https://github.com/Simple-Robotics/proxsuite/pull/261

## New Contributors
* @costashatz made their first contribution in https://github.com/Simple-Robotics/proxsuite/pull/255

**Full Changelog**: https://github.com/Simple-Robotics/proxsuite/compare/v0.4.1...v0.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Simple-Robotics/proxsuite/releases/tag/v0.5.0'>ProxSuite 0.5.0</a>.</em>

**Comment by stephane-caron** [comment]
This automated release note is quite underwhelming: ProxSuite 0.5.0 introduces [**support for nonconvex QPs**](https://github.com/Simple-Robotics/proxsuite/issues/237), that's a new feature! It should be stated plainly.

(Edit: updated the release note.)

Like changelogs, release notes are _for humans_, not machines. (Borrowed from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).) For instance they can be subscribed to to follow new features and changes in a project at release granularity. Automated release notes have some benefits too, but shouldn't we rather _append_ them below a message written by humans for humans?
